scripts/makebumpver

```diff
@@ -328,6 +328,9 @@ class MakeBumpVer:
                 continue
 
             if self.rhel:
+                # First place where we need to use JIRA server, don't require it until now
+                self._jira.connect()
```

Comment:
Isn't it s/self._jira/self.jira?

Reply:
Oh ignore I see now.

Reply:
Probably should be but let's not complicate things here. :)